dusty/commands/setup.py

```diff
@@ -26,12 +28,35 @@ def _get_default_specs_repo():
     _pretty_print_key_info(constants.CONFIG_SPECS_REPO_KEY)
     return _get_raw_input('Input the full name of your specs repo, e.g. github.com/gamechanger/example-dusty-specs: ')
 
+def _get_contents_of_file(file_location):
+    with open(file_location) as f:
+        return f.readlines()
+
+def _append_to_file(file_location, new_line):
+    with open(file_location, 'a') as f:
+        f.write(new_line)
+
+def _setup_nginx_config(nginx_conf_location):
+    include_path = '{}/servers'.format(nginx_conf_location)
+    if not isdir(include_path):
+        mkdir(include_path)
+    _append_to_file('{}/nginx.conf'.format(nginx_conf_location), 'include servers/*;')
```

Comment:
May want to surround this in newlines to be safe in case they don't already have a newline at the end of the file